app/src/main/res/layout/problem_detail.xml

```diff
@@ -1,4 +1,5 @@
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
`problem_detail.xml` and `problem_detail_ref.xml` are nearly identical. Also `problem_detail_ref.xml` seems unused. Feel free to remove `problem_detail_ref.xml`.